curvefs/src/mds/topology/topology_metric.h

```diff
@@ -187,6 +187,22 @@ class TopologyMetricService {
     TopologyOption option_;
 };
 
+struct FsMetric {
+ public:
+    const std::string kTopologyFsMetricPrefix = "topology_Fs_id_";
+    // inode number
+
+    bvar::Status<uint64_t> inodeNum_;
+
+    explicit FsMetric(FsIdType fsId)
+        : inodeNum_(kTopologyFsMetricPrefix,
+                    std::to_string(fsId) + "_inode_num", 0) {}
+};
+
+using FsMetricPtr = std::unique_ptr<FsMetric>;
+
+extern std::map<FsIdType, FsMetricPtr> gFsMetrics;
```

Comment:
there is no need to put a declaration here because no other translation unit uses it.

Reply:
> there is no need to put a declaration here because no other translation unit uses it.

fix